Missionframework/modules/01_core/fnc/fn_core_createFobMarker.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+    KPLIB_fnc_core_createFobMarker
+
+    File: fn_core_createFobMarker.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-05-12
+    Last Update: 2018-05-12
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+    Creates FOB marker
+
+    Parameter(s):
+        0: ARRAY - Position where FOB has to be built.
+
+    Returns:
+    STRING - Marker name
+*/
+params ["_position"];
+
+private _index = count KPLIB_sectors_fobs;
```

Comment:
I think this could be an issue later.
Imagine you've 5 FOBs -> index is 5, so it creates KPLIB_fob_5. This is fine so far and now an index would be 6 due to the count.
Now the player loose FOB 2 and creates a new one. Count returns 5 and it would be another KPLIB_fob_5.

Reply:
I think you're right concerning just store the positions for the FOBs. This also eliminates the need to have the getMarkerPosition conversion.